src/python/foglamp/core/scheduler/scheduler.py

```diff
@@ -662,7 +663,7 @@ def _schedule_first_task(self, schedule, current_time):
             },
             "max_completed_task_age_days": {
                 "description": "The maximum age, in days (based on the start time), for a rows "
-                               "in the tasks table that do not have a status of 'running'",
```

Comment:
@MarkRiddoch Note that having a single quoted string 'running' also caused a crash.

Reply:
Let's remove the quotes today and raise a Jira to get this fixed in the next sprint